### PR TITLE
Ensure unit tests pass with any Node.js ABI version

### LINKED ISSUE #113: Tests are broken on node 12, add to travis
Something to do with `nock`?

### PR DESCRIPTION
**Pull request by lovell**
Fixes #113

* Pins to ABI 64 as this is supported by the `a-native-module` dependency
* Adds Node.js 12 to CI environment
* Node.js 14 is not yet supported by the `verify-travis-appveyor` dependency

### CODE REVIEW

**Comment by vweevers** [comment]
We can remove `verify-travis-appveyor`. In the 2 years that we used it (here and elsewhere), it never barked. In other words, we didn't make the tiny mistakes that this tool intents to prevent.

**Comment by lovell** [comment]
Thanks, have updated to remove `verify-travis-appveyor` and add Node.js 14 to the CI matrix.

**Comment by vweevers** [comment]
Retried node 14 on AppVeyor a bunch of times, keeps failing with exit code `1073741819` on either `npm install` or `npm test`:

```
npm test
Command exited with code -1073741819
```

**Comment by lovell** [comment]
Appveyor's x86 Node.js 14 environment has been failing about 50% of the time for other repos over the last couple of days so I don't think it is related.

**Comment by vweevers** [comment]
Link dump:

https://github.com/appveyor/ci/issues/3407 (same issue but on an earlier 14.x)
https://github.com/FeodorFitsner/node-14-x86-issue (repro)
https://github.com/npm/cli/issues/1227 (npm report)
https://github.com/nodejs/node/issues/32863 (possibly related)

File: appveyor.yml [hunk 1]
```diff
@@ -13,6 +13,8 @@ version: "{build}"
 environment:
   # Test against these versions of Node.js
   matrix:
+    - nodejs_version: "14"
```
Comment (vweevers) [inline appveyor.yml:16]:
```suggestion
    - nodejs_version: "14.2.0"
```

It seems this version doesn't have the bug (an access violation in V8 on GC, it looks like).

**Pull request merged**